templates/docs/rhpam77-authoring-ha.adoc

```diff
@@ -66,6 +66,7 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
 |`KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED` | `KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED` | If set to true, turns on KIE server global discovery feature (Sets the org.kie.server.controller.openshift.global.discovery.enabled system property) | false | False
 |`KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | `KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | Enables connection to KIE Server via OpenShift internal Service endpoint. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property) | true | False
 |`KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | `KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | KIE ServerTemplate Cache TTL in milliseconds. (Sets the org.kie.server.controller.template.cache.ttl system property) | 5000 | False
+|`KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` | -- | true
```

Comment:
This line has broken the table. The table has 5 columns where the line only has 3 here. Also, the description should say "If set to true, turns on OpenShift integration feature (Sets the org.kie.server.controller.openshift.enabled system property)".

Reply:
I'll remove and leave the adoc update procedure to update

Reply:
@Sgitario @spolti Will those adoc items being updated automatically by the build?